app/views/docs/index.phtml

```diff
@@ -76,15 +76,15 @@ $cols = [
                     <li><a href="/docs/databases">Databases</a></li>
```

Comment:
I think the realtime link should be at the top or bottom. We should sort the services in the same order they appear on the console.